scalability_and_performance/optimization/optimizing-networking.adoc

```diff
@@ -14,14 +14,15 @@ Geneve encapsulates all tunneled traffic in user datagram protocol (UDP) packets
 inner-packets are subject to normal checksumming rules to guarantee data is not corrupted during transit. Depending on CPU performance, this additional
 processing overhead can cause a reduction in throughput and increased latency when compared to traditional, non-overlay networks.
 
-Cloud, VM, and bare metal CPU performance can be capable of handling much more than one Gbps network throughput. When using higher bandwidth links such as 10 or 40 Gbps, reduced performance can occur. This is a known issue in Geneve-based environments and is not specific to containers or {product-title}. Any network that relies on Geneve or VXLAN tunnels will perform similarly because of the tunnel implementation.
+Cloud, virtual, and bare-metal environments running {product-title} can achieve multi-gigabit network throughput. Production clusters using OVN-Kubernetes with Geneve tunneling can handle high-throughput traffic effectively without changes or special configuration. 
 
-If you are looking to push beyond one Gbps, you can:
+In some high-performance scenarios where maximum efficiency is critical, targeted performance tuning can help optimize CPU usage, reduce overhead, and ensure that you are making full use of the NIC's capabilities.
 
-* Evaluate network plugins that implement different routing techniques, such as border gateway protocol (BGP).
```

Comment:
@mcurry-rh Maybe the use of network-plugins can be changed to :

` Evaluate OVN-Kubernetes User Defined Networking (UDN) routing techniques, such as border gateway protocol (BGP).`

Reply:
@jtaleric @mcurry-rh @venkataanil Added this information.